kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/client/messages/Messages.properties

```diff
@@ -562,6 +562,7 @@ netHwRadio=Radio
 netHwBand=Frequency Band
 netHwLAC=LAC
 netHwCI=CI
+netHwWlanChannel=WLAN Channel
```

Comment:
As in comment above, I would use a more menaningful name. Remember to update also the japanese version (just use google translate, I will ask for review later).